### PR TITLE
Revert autocommitgenericsql

### PR DESCRIPTION
**Pull request by yucohen**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
reverted the autocommit from the code and added it as true by default for ms sql instead.
added the option to disable it by setting to false

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/GenericSQL/ReleaseNotes/1_0_25.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{"breakingChanges":true,"breakingChangesNotes":"The ***Connection Arguments*** parameter now specifies 'autocommit=True' by default."}
```
Comment (DeanArbel) [inline Packs/GenericSQL/ReleaseNotes/1_0_25.json:1]: No need for this - it won't affect existing users.

**Review by DeanArbel — APPROVED** [review]

**Review by DeanArbel — APPROVED** [review]

**Comment by ShirleyDenkberg** [comment]
@DeanArbel @glicht Doc review completed.

File: Packs/GenericSQL/Integrations/GenericSQL/GenericSQL.py [hunk 2]
```diff
@@ -69,6 +69,7 @@ def parse_connect_parameters(connect_parameters: str, dialect: str, verify_certi
             connect_parameters_dict[key] = value
         if dialect == "Microsoft SQL Server":
             connect_parameters_dict['driver'] = 'FreeTDS'
+            connect_parameters_dict.setdefault('autocommit', 'True')
```
Comment (glicht) [inline Packs/GenericSQL/Integrations/GenericSQL/GenericSQL.py:72]: Do this also for the dialect: Microsoft SQL Server - MS ODBC Driver

**Comment by yucohen** [comment]
@DeanArbel Requires force merge due to failing validation regarding docker image update

**Review by DeanArbel — APPROVED** [review]

**Pull request merged**